lib/src/inline_syntaxes/decode_html_syntax.dart

```diff
@@ -22,14 +21,16 @@ class DecodeHtmlSyntax extends InlineSyntax {
     if (parser.pos > 0 && parser.charAt(parser.pos - 1) == $backquote) {
       return false;
     }
+
     final match = pattern.matchAsPrefix(parser.source, parser.pos);
-    if (match == null ||
-        match[1] != null ||
-        (match[2] == null && match[3] == null)) {
+    if (match == null) {
       return false;
     }
 
-    if (match[1] != null && htmlEntitiesMap[match.match] == null) {
+    // TODO(Zhiguang): Enable HTML entity decoding when working on HTML escape
+    // issues.
+    final isEntity = match[1] != null;
+    if (isEntity) {
```

Comment:
Eh...don't think this helps. Just putting the `match[1] != null` in the `if` block is cleaner

Reply:
Yep, fixed.

I will give them meaningful names later when it becomes confusing. :)